server/www/templates/posts/media.html

```diff
@@ -1,18 +1,38 @@
+<form>
+    <div class="form-field file">
 
-<form ng-submit="uploadFile()">
-    <div class="form-field">
-      <a ng-href="{{media.original_file_url}}" ng-if="media.original_file_url" target="_blank" translate>post.media.current_url</a>
+        <!-- the image file -->
+        <figure ng-if="media.original_file_url" >
+            <img src="{{media.original_file_url}}" />
```

Comment:
This needs a width or height param otherwise a large photo takes up the whole screen...